File: tests/fixtures/sample-docs.kt
/**
 * User authentication service.
 */
class AuthService {
    /**
     * Validate a JWT token and return the decoded payload.
     */
    fun validateToken(token: String): Map<String, Any> {
        return emptyMap()
    }

    fun revokeToken(token: String): Boolean {
        return true
    }
}

File: tests/fixtures/sample-docs.php
<?php
/**
 * User authentication service.
 */
class AuthService
{
    /**
     * Validate a JWT token and return the decoded payload.
     */
    public function validateToken(string $token): array
    {
        return [];
    }

    public function revokeToken(string $token): bool
    {
        return true;
    }
}
